Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/BaseClasses/package.mo

```diff
@@ -0,0 +1,11 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.InductionMotors;
+package BaseClasses "Package with base classes for MotorDrive.InductionMotors"
+  extends Modelica.Icons.BasesPackage;
+
+
```

Comment:
remove empty lines.

Reply:
Thank you for your review, I have removed the unnecessary empty lines in all files.